ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
 
-## Steps to fork the repo: 
+## Steps to fork the repository (repo): 
 
 1. Go to the main GitHub page of the repo:
     (https://github.com/gbowne1/codebooker/)
    
-1. Once you are on the main GitHub page of the repo, click the **Fork** button at the top right of the page. 
+1. Select **Fork** at the top right of the page. 
```

Comment:
Because we already told the user to go to the main page of the repo, we don't need to repeat that information. Numbers indicate the steps must happen in order. 

Side note: If we were to keep that introductory dependent clause, we should change "once" to "after" for precise language. "Once" can mean it only happens one time, which can be confusing for users for whom English isn't their first language. It can also be confusing for translators if your docs are being translated into other languages.

I changed "click" to "select" to reflect many modern style guides, which are trying to include tablets and phones. "Click" isn't wrong but is sometimes seen as outdated.

Because bolding indicates it's part of the UI, most software style guides I've used say not to specify that it's a button, so I removed that part. Some technical writers will say you shouldn't explain where the button is unless it's really hard to find, but I left that in.